script.js

```diff
@@ -0,0 +1,46 @@
+const mobileMenu = document.createElement('div');
+function hideMobileMenu() {
+  mobileMenu.style.display = 'none';
+}
+function showMobileMenu() {
+  mobileMenu.style.display = 'block';
+}
+mobileMenu.className = 'mobile-menu';
+mobileMenu.innerHTML = '<img></p><ul class="menu"><li><a>Portfolio</a></li><li><a>About</a></li><li><a>Contact</a></li></ul>';
+document.body.appendChild(mobileMenu);
+mobileMenu.style.display = 'none';
+mobileMenu.style.paddingTop = '34px';
+mobileMenu.style.boxSizing = 'border-box';
+mobileMenu.style.position = 'absolute';
+mobileMenu.style.top = '0px';
+mobileMenu.style.left = '-8px';
+mobileMenu.style.backgroundColor = '#3c3a39';
+mobileMenu.style.height = '755px';
```

Comment:
- [ ] **[REQUIRED CHANGES]**

- Kindly, make the height of  `.mobile-menu` 100% to take up the whole device height making sure the user must click on any link before seeing the contents of your page. As of right now, if a user decides not to click any of the menu links, they can ignore the content on your mobile menu and scroll down your page, having all the contents of your page at their disposal. So why add a mobile menu content when I could still see what I want to see on scrolling?
Please, kindly improve your design to fix that.

Reply:
- [ ] **[REQUIRED CHANGES]**

- Please, kindly improve your design to match the Figma design. Your design is missing the  `black line` and also, I think your `.mobile-menu` should have a parent container so that the parent container could have a white background and the `black line`, a height and width of `100%` so that its child `.mobile-menu` can take up 90% of the height from its parent and uses flex-box to layout the content in your `.mobile-menu`.

```

.parent-container {
  background: #fff;
  height: 100%;
  width: 100%;
  // position: fixed;
 top, right, bottom, left
}

.mobile-menu {
  width: 100%;
  height: 90%;
  background: #3c3a39;
}

```
> Please, compare your design with the figma design to improve it.

**Figma**

![hey](https://user-images.githubusercontent.com/59397159/167457291-55cae653-10bf-4ff5-a028-2a4a7bac8221.png)